### PR TITLE
Bug 1134722: Use aries instead of shinano manifest file.

### PR DESCRIPTION
**Pull request by walac**
This allows mozharness to build aries device. Mozharness takes the
manifest from the gecko tree, and expects the manifest file name be the
same as the target name.

### CODE REVIEW

File: config.sh [hunk 1]
```diff
@@ -171,7 +171,7 @@ case "$1" in
 	echo DEVICE=vixen >> .tmp-config &&
 	echo PRODUCT_NAME=vixen >> .tmp-config &&
 	repo_sync $1
-	;;  
+	;;
```
Comment (lissyx) [inline config.sh]: nit: unrelated change

Comment (walac) [inline config.sh]: Ah, sorry, this is from my vim configs.
